SIGS.md

```diff
@@ -0,0 +1,184 @@
+
+### SIG Autoscaling
+
+> **👥 Leadership:** Tamar Eilam, Abhishek Malvankar
+
+**Charter**: Develop intelligent autoscaling solutions that automatically adjust llm-d deployments based on traffic patterns, workload characteristics, and hardware utilization.
+
+**Key Areas**:
+- Traffic-aware autoscaling algorithms
+- Hardware-specific scaling policies
+- Workload-based capacity planning
+- Integration with Kubernetes HPA/VPA
+- Cost-optimized scaling strategies
+
+### SIG Observability
+
+> **👥 Leadership:** Sally O'Malley
+
+**Charter**: Provide comprehensive monitoring, logging, and observability capabilities that enable operators to understand system behavior, diagnose issues, and optimize performance.
+
+**Key Areas**:
+- Metrics collection and visualization
+- Distributed tracing and logging
+- Performance monitoring and alerting
+- Operational dashboards and reporting
+- Integration with monitoring ecosystems (Prometheus, Grafana, etc.)
+
+## Getting Involved
+
+### Joining a SIG
+1. **Attend a meeting**: Check the [project calendar](https://red.ht/llm-d-public-calendar) for SIG meeting times
+2. **Join the conversation**: Participate in SIG-specific channels on [Slack](https://inviter.co/llm-d-slack)
+3. **Review documentation**: Read the SIG's charter and current initiatives
+4. **Start contributing**: Look for "good first issues" labeled with the SIG's area
+
+### SIG Communication Channels
+- **Slack**: Each SIG has dedicated channels in the [llm-d Slack workspace](https://llm-d.slack.com)
+- **Google Groups**: Join [llm-d-contributors](https://groups.google.com/g/llm-d-contributors) for comment access to SIG documents
+- **GitHub**: Issues and discussions are labeled by SIG area
+- **Calendar**: All SIG meetings are on the [shared project calendar](https://red.ht/llm-d-public-calendar)
+
+## SIG Formation and Evolution
+
+### Creating a New SIG
+1. **Identify need**: Demonstrate community interest and technical necessity
+2. **Draft charter**: Define scope, goals, and initial leadership
+3. **Proposal process**: Submit proposal following [project contribution guidelines](CONTRIBUTING.md)
+4. **Community review**: Present at weekly project standup and gather feedback
+5. **Approval**: Obtain approval from project maintainers
+
+### SIG Lifecycle Management
+- **Active**: Regular meetings, active development, engaged community
+- **Maintenance**: Limited active development, focus on stability and bug fixes
+- **Archived**: No longer active, historical reference only
+
+SIGs may evolve, merge, or be archived based on project needs and community engagement.
+
+## Resources
+
+- **Project Calendar**: [llm-d Public Calendar](https://red.ht/llm-d-public-calendar)
+- **Slack Workspace**: [https://llm-d.slack.com](https://llm-d.slack.com)
+- **Google Groups**: [https://groups.google.com/g/llm-d-contributors](https://groups.google.com/g/llm-d-contributors)
+- **Project Overview**: [PROJECT.md](PROJECT.md)
```

Comment:
This feels like contributing guide or community governance instead of project overview.